lib/media/drm_engine.js

```diff
@@ -1807,13 +1807,40 @@ shaka.media.DrmEngine = class {
       'com.apple.fps',
     ];
 
+    const widevineRobustness = [
+      'SW_SECURE_CRYPTO',
+      'SW_SECURE_DECODE',
+      'HW_SECURE_CRYPTO',
+      'HW_SECURE_DECODE',
+      'HW_SECURE_ALL',
+    ];
+
+    const playreadyRobustness = [
+      '150',
+      '2000',
+      '3000',
+    ];
+
+    const testRobustness = {
+      'com.widevine.alpha': widevineRobustness,
+      'com.microsoft.playready': playreadyRobustness,
+      'com.microsoft.playready.recommendation': playreadyRobustness,
+      'com.chromecast.playready': playreadyRobustness,
```

Comment:
Only recommendation key system supports checking robustness. Legacy key systems ignore it, so to check SL3000 you need to use `com.microsoft.playready.hardware` key system instead.

Reply:
Actually that only applies to Edge and Xbox, right?

Reply:
Recommendation is available for sure on Edge, Xbox and PS5.

Reply:
Ok, please create a PR with this change. Thanks!